blog.css

```diff
@@ -0,0 +1,63 @@
+body {
+  font-family: 'Josefin Slab', serif;
+  font-size: 1.2rem;
+}
+
+/* Header */
+header {
+  background-image: url(images/header-image.jpg);
+}
+
+header img {
+  max-width: 35%;
+  height: auto;
+  /* float: left; */
+}
+
+header p {
+  color: green;
+  font-size: 1.5rem;
+  font-style: italic;
+  font-weight: bold;
+}
+
+.checkout-page-link {
+  color: white;
+  float: right;
+}
+
+.menu-bar {
+  background: lightgrey;
+  padding: 1px;
+}
+
+li {
+  display: inline;
+}
+
+a {
+  color: darkblue;
+  text-decoration: none;
+  font-weight: bold;
+}
+
+.blog1, .blog2, .blog3 {
```

Comment:
You can use one class across all three blogs for styling. You don't need to create a class for each new blog if they are teh same.